code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -65,6 +65,8 @@ GLOBAL_DATUM_INIT(pipe_icon_manager, /datum/pipe_icon_manager, new())
 // Icons/overlays/underlays
 /obj/machinery/atmospherics/update_icon()
 	var/turf/T = loc
+	if(!istype(T)) // Somehow loc became null

```

Comment:
might be a good idea to qdel it if it's loc is null

Reply:
Can't think of a situation where it should still exist so yeh good one

Reply:
Maybe check if its already qdeleted before qdeling it.
Also... can there be situations where loc is not a turf, but also not null?
For example, if someone is transporting a stack of pipes or other atmos machinery unwrenched, inside a locker?